README.md

```diff
@@ -1,15 +1,15 @@
-# Centreon Broker
+# Centreon Collect
 
-Centreon Broker is an extensible open-source monitoring event
-transmitter (broker). It is a low-level component of the
+Centreon Collect is an extensible open-source monitoring event
```

Comment:
This move is a good idea but you cannot change broker to collect everywhere :smile: 
I think it is a little more difficult...

Reply:
Maybe a mix of engine and broker README files would be a good point.